src/frontend/src/binder/bind_context.rs

```diff
@@ -90,7 +90,7 @@ impl BindContext {
             .get(column_name)
             .ok_or_else(|| ErrorCode::ItemNotFound(format!("Invalid column: {}", column_name)))?;
         if columns.len() > 1 {
-            Err(ErrorCode::InternalError("Ambiguous column name".into()).into())
+            Err(ErrorCode::InternalError(format!("Ambiguous column namw: {}", column_name)).into())
```

Comment:
I'm not sure if it's worth making a new Ambiguous error since there only seems to be 2 instances of ambiguous errors being thrown in the code.

That being said, I think it's a better solution to throw errors for any non-`ItemNotFound` errors instead of only throwing errors for `InternalError`.

Reply:
FYI, we will eventually introduce a separate `BindError` and refactor all of the internal errors in the binder. So I believe it's acceptable for now. #2366